cmd/admin/main.go

```diff
@@ -23,6 +23,13 @@ import (
 	"github.com/apache/dubbo-admin/pkg/admin/services"
 )
 
+// @title           Dubbo-Admin API
+// @version         1.0
+// @description     This is a dubbo-admin swagger ui server.
+// @license.name  Apache 2.0
+// @license.url   http://www.apache.org/licenses/LICENSE-2.0.html
+// @host      127.0.0.1:38080
```

Comment:
Port should be set as 38081?

Reply:
i just pick 38081 to avoid port conflict,   any suggestion?

Reply:
It's okay. If there is a conflict in the future, let's take it into account.